template/build/vue-loader.conf.js

```diff
@@ -15,5 +15,11 @@ module.exports = {
     source: 'src',
     img: 'src',
     image: 'xlink:href'
+  },
+  buble: {
+    objectAssign: 'Object.assign',
+    transforms: {  // turn off the `with` removal
+      stripWith: false
```

Comment:
That has to go definitely, it's not required to make object rest spread work - to the contrary, it would probably breka most builds using ES modules in webpack 2 and upwards.

Reply:
I don't understand. Are you saying that the buble block here should NOT be added because it would break most builds? I thought this was what you were suggesting [here](https://github.com/vuejs-templates/webpack/issues/1005#issuecomment-340443636) to make that issue work.

Reply:
OK I just read your comment [here](https://github.com/vuejs-templates/webpack/issues/1005#issuecomment-343876559). Is this why you're saying that adding this will break most builds?

It seems to suggest that projects built with recent version of the vuejs webpack template should not break. But it is breaking with the current version of the template, at least it was at the time I made [that issue](https://github.com/vuejs-templates/webpack/issues/1005).

Reply:
What I mean is that this is the option we want:
```
objectAssign: 'Object.assign',
```
and this is the option we don't want:
```
transforms: {  // turn off the `with` removal
+      stripWith: false
```